src/Components/Artist/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.test.tsx

```diff
@@ -25,12 +25,15 @@ describe("CollectionsRail", () => {
 
   it("Renders expected fields", () => {
     const component = getWrapper()
-    expect(component.text()).toMatch("Browse by iconic collections")
-    expect(component.find(ArtistCollectionEntity).length).toBe(8)
-    expect(component.text()).toMatch("Flags")
-    expect(component.text()).toMatch("From $1,000")
-    expect(component.text()).toMatch("Street Art Now")
-    expect(component.text()).toMatch("From $200")
+    // Settimeout needed here for carousel render
+    setTimeout(() => {
```

Comment:
Did the `renderUntil` enzyme helper not work that I linked yesterday? The idea is that enzyme will render and then wait until a certain class is on the dom before returning the wrapper that one then uses to make assertions against. 

If you do a global search in Reaction for `renderUntil` there are a lot of examples, [such as this](https://github.com/damassi/reaction/blob/a5e1c0b47091dc595a4cf557c5618d870992e593/src/Apps/Search/Routes/Artworks/Components/Filter/__tests__/SearchResultsArtworkGrid.test.tsx#L13). 

As an aside, `setTimeout` can often be swapped with `setImmediate` -- which is the same thing as `setTimeout` with a time set to 0. This means "on the next run loop, execute this function" which is often all that's needed. But avoiding this sort of run-loop hack is specifically why `renderUntil` was written. Let me know if the examples in Reaction aren't clear.

Reply:
I would have preferred to use the `renderUntil` but I didn't have any luck getting it to work properly, so I used the `setTimeout` which fixed the issue.

Reply:
I'm down to pair, and others in the office have used it; they'll be able to help if need be.

Reply:
Just resolved this.